default.config.js

```diff
@@ -61,6 +61,36 @@ module.exports = {
       pipeWarnTime: 500,
       pipeTimeout: 5000,
       initTimeout: 10000,
+    },
+    'kuzzle-plugin-logger': {
```

Comment:
shouldn't this be in a custom kuzzlerc configuration file in docker images, especially considering that we don't always ship these plugins?

Reply:
I putted it in default config not because they might be shipped but because they are officials. So, don't you think officials plugins codes should be directly here ?

Reply:
I don't think they should be there.
1. do all these plugins throw errors with codes?
2.why should we version error subcodes for plugins in the kuzzle core repository?
3. if we _do_ want to assign arbitrary numbers for these plugins, we should do that only in our packages (e.g. in our docker files), using custom configuration files.